README.md

```diff
@@ -216,6 +216,8 @@ If any value is set for `NHAIS_MONGO_HOST` then the following properties will be
 | NHAIS_MONGO_USERNAME             |         | (Optional) Mongodb username. If set then password must also be set.
 | NHAIS_MONGO_PASSWORD             |         | (Optional) Mongodb password
 | NHAIS_MONGO_OPTIONS              |         | (Optional) Mongodb URL encoded parameters for the connection string without a leading ?
+| NHAIS_MONGO_TTL                  | P30D    | (Optional) Time-to-live value for inbound and outbound state collection documents in ISO-8601 format
```

Comment:
Let's be more specific here and include a reference to avoid any confusion

`... documents as an [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations)`